salt/modules/saltutil.py

```diff
@@ -945,6 +947,35 @@ def clear_cache():
     return True
 
 
+def clear_job_cache(hours=24):
+    '''
+    Forcibly removes job cache folders and files on a minion.
+
+    .. versionadded::
+
+    WARNING: The safest way to clear a minion cache is by first stopping
+    the minion and then deleting the cache files before restarting it.
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' saltutil.clear_job_cache hours=12
+    '''
+    threshold = time.time() - hours * 3600
+    for root, dirs, files in salt.utils.safe_walk('{0}{1}minion_jobs'.format(__opts__['cachedir'], os.sep),
+                                                  followlinks=False):
+        for name in dirs:
+            try:
+                mtime = os.path.getmtime(os.path.join(root, name))
+                if mtime < threshold:
+                    shutil.rmtree(os.path.join(root, name))
+            except OSError as exc:
+                log.error('Attempt to clear cache with saltutil.clear_job_cache FAILED with: {0}'.format(exc))
```

Comment:
``str.format()`` is overkill in logging, since the logging module already performs string replacement. This should be modified as follows:

```python
log.error('Attempt to clear cache with saltutil.clear_job_cache FAILED with: %s', exc)
```